programs/voter-stake-registry/src/instructions/close_voter.rs

```diff
@@ -97,31 +111,42 @@ pub fn close_voter<'info>(ctx: Context<'_, '_, '_, 'info, CloseVoter<'info>>) ->
 
         let voter_seeds = voter_seeds!(voter);
 
-        // will close all the token accounts owned by the voter
-        for deposit_vault_info in ctx.remaining_accounts {
-            let deposit_vault_ta = Account::<TokenAccount>::try_from(deposit_vault_info)
-                .map_err(|_| MplStakingError::DeserializationError)?;
-            registrar.voting_mint_config_index(deposit_vault_ta.mint)?;
+        let calculated_atas_to_close = filtered_mints.into_iter().map(|voting_mint_config| {
+            get_associated_token_address(&ctx.accounts.voter.key(), &voting_mint_config.mint)
+        });
 
+        for (index, calculated_ata_to_close) in calculated_atas_to_close.enumerate() {
+            require!(
```

Comment:
We've already checked this before. What might have changed?

Reply:
Tunnel vision. Thanks!